src/qibolab/_core/components/configs.py

```diff
@@ -43,6 +46,22 @@ class DcConfig(Config):
 
     offset: float
     """DC offset/bias of the channel."""
+    filters: Filters = Field(default_factory=list)
+    """List of filters."""
+
+    @property
+    def feedback(self) -> list[float]:
+        feedback_coeff = [i.feedback for i in self.filters if i is not None]
+        if len(feedback_coeff) > 0:
+            return reduce(np.convolve, feedback_coeff)
+        return []
+
+    @property
+    def feedforward(self) -> list[float]:
+        feedforward_coeff = [i.feedforward for i in self.filters if i is not None]
+        if len(feedforward_coeff) > 0:
+            return reduce(np.convolve, feedforward_coeff)
+        return []
```

Comment:
Unfortunately, here there is no neutral element. There would be if $a_0$ was taken into account, but I'm pretty sure that it is not.

So, just reverse the condition
```suggestion
        if len(feedforward_coeff) > 0:
            return []
        return reduce(np.convolve, feedforward_coeff)
```